dialogue-core/src/main/java/com/palantir/dialogue/core/RetryingChannel.java

```diff
@@ -103,18 +154,18 @@ private RetryingCallback(
 
         ListenableFuture<Response> failure(Throwable throwable) {
             if (++failures <= maxRetries) {
-                logRetry(throwable);
-                return execute();
+                return retry(throwable);
             }
             return Futures.immediateFailedFuture(throwable);
         }
 
-        private void logRetry(Throwable throwable) {
+        private void logRetry(long backoffNanoseconds, Throwable throwable) {
             if (log.isInfoEnabled()) {
                 log.info(
                         "Retrying call after failure",
                         SafeArg.of("failures", failures),
                         SafeArg.of("maxRetries", maxRetries),
+                        SafeArg.of("backoffNanoseconds", backoffNanoseconds),
```

Comment:
could we do a TimeUnit.Milliseconds.convert here and present backoffMillis instead? It's really hard to eyeball such big numbers when they appear in the logs

Reply:
good call